### PR TITLE
Added Myself

### PR DESCRIPTION
**Pull request by ApolloDev0**
had a few issues but i think it should be working this time!

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -406,5 +406,12 @@
     "githubUsername": "DenverCoder1",
     "imageUrl": "https://user-images.githubusercontent.com/20955511/103463192-546c8100-4d33-11eb-9197-fec4c5396e27.png",
     "issueNumber": 209
+  },
+    {
```
Comment (AllanRegush) [inline data.json:410]: Could you adjust your formating the rest of the entries? Thanks 👍

**Review by AllanRegush — COMMENTED** [review]
Thank for your PR 👍 Just a few adjustments.

File: data.json (hunk 1 above)
Comment (ApolloDev0) [reply data.json:410]: im sorry could you explain more?

Comment (eddiejaoude) [reply data.json:410]: The indentation does not align consistently

Comment (ApolloDev0) [reply data.json:410]: oh i see okay i'll try to fix it

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**